bnd-maven-plugin/.gitignore

```diff
@@ -1 +1,4 @@
+/.classpath
+/.project
+/.settings/
```

Comment:
I'll add my metadata (generated by m2e) and see if that is to everyone's liking.